cmd/catalyst-node/catalyst-node.go

```diff
@@ -5,23 +5,29 @@ import (
 	"errors"
 	"flag"
 	"fmt"
+	"github.com/hashicorp/serf/cmd/serf/command/agent"
```

Comment:
Hm, looks like our autoformatters disagree about where this line should be.

Reply:
interesting. Updated back in 45710a545b44d9d636b16714b8055000286d6f92